SocialFilter/sections/specific.txt

```diff
@@ -77,6 +77,7 @@ mantan-web.jp,fufukokan.af-original.com,myoji-yurai.net,connpass.com,journals.as
 !
 blog.unity.com##.sm\:flex-shrink-0 > div[class=""]
 visual.republika.co.id##div[id^="live_room"]
+blog.eldorado.ru##.repost, .js-repost
```

Comment:
We rarely use multiple selectors.
I guess just using this rule is enough.
```adblock
blog.eldorado.ru##.repost
```

Reply:
Yes, it works, thanks!